src/client/components/Message/Message.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './Message.css';
+import PropTypes from 'prop-types';
+
+// message:{text, senderId,}, userId
+// Was the message sent by the current user. If so, add a css class
+// here message is object which contains user which need not be logged in and the text message
+const Message = ({ message, username }) => {
+  return (
+    <div className="chat-message__body">
+      <p
+        className={`chat-message__message ${message.name === username &&
+          'chat-message__receiver'}`}
+      >
+        {message.message}
```

Comment:
Why does the message have a key called message? 

The message should just take message that is a string and username that is also a string